packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -163,7 +164,10 @@ abstract class StreamBase extends Resource implements IStream {
     return ret;
   }
 
-  private grant(grantee: iam.IGrantable, ...actions: string[]) {
+  /**
+   * Grant the indicated permissions on this key to the given IAM principal (Role/Group/User).
```

Comment:
Why do you say `permissions on this key`, rather than `permissions` on this stream?

Reply:
updated